Palace/Utilities/UI/UIApplication+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+@objc extension UIApplication {
+  static var darkModeEnabled: Bool {
+    if #available(iOS 13, *) {
+      return UITraitCollection.current.userInterfaceStyle == .dark
```

Comment:
Probably we can avoid this global variable by adding a color, compatible with iOS versions prior to 13, to UIColor, something like `compatibleTextColor` (just an example) that returns .label/asset color for iOS 13+ and black for other versions, as we are not really interested in dark mode value itself.